### ISSUE TITLE
versions required in setup.py if CI in OS.environ get through to pypi.org metadata

### ISSUE THREAD

**Issue by zdaq12** [atmos-cloud-sim-uj/PySDM#884]

**Comment by slayoo** [comment]
Confirming, the `...dist-info/METADATA` file constains:
```
Requires-Dist: ThrustRTC (==0.3.19)
Requires-Dist: CURandRTC (==0.1.6)
Requires-Dist: numba (==0.55.2)
Requires-Dist: numpy (==1.21)
Requires-Dist: Pint (==0.17)
Requires-Dist: chempy (==0.7.10)
Requires-Dist: scipy (==1.6.3)
Requires-Dist: pyevtk (==1.2.0)
```
which is likely resulting from building the package before upload on GitHub Actions where the CI env var is set

**Comment by slayoo** [comment]
hopefully addressed in #885 and in the v2.11 release, let's see soon if it helped!

**Comment by slayoo** [comment]
fixed in v2.11:
```
Requires-Dist: ThrustRTC (==0.3.20)
Requires-Dist: CURandRTC (>=0.1.2)
Requires-Dist: numba (>=0.51.2)
Requires-Dist: numpy
Requires-Dist: Pint
Requires-Dist: chempy
Requires-Dist: scipy
Requires-Dist: pyevtk
```
